builder/converter/templates/categorized_view_template.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<org.jenkinsci.plugins.categorizedview.CategorizedJobsView plugin="categorized-view@1.9">
+    <name></name>
+    <filterExecutors>false</filterExecutors>
+    <filterQueue>false</filterQueue>
+    <properties class="hudson.model.View$PropertyList"/>
+    <jobNames>
+        <comparator class="hudson.util.CaseInsensitiveComparator"/>
+    </jobNames>
+    <jobFilters/>
+    <columns>
+        <hudson.views.StatusColumn/>
+        <hudson.views.WeatherColumn/>
+        <org.jenkinsci.plugins.categorizedview.IndentedJobColumn/>
+        <hudson.views.LastSuccessColumn/>
+        <hudson.views.LastFailureColumn/>
+        <hudson.views.LastDurationColumn/>
+        <hudson.views.BuildButtonColumn/>
+    </columns>
+    <includeRegex></includeRegex>
+    <recurse>true</recurse>
+    <groupingRules/>
```

Comment:
I believe that these elements are processed like the ones we can found on a standard list view, so that's why I haven't implemented specific tests.

I'll find a way to link "list view" unit tests to "categorized view" ones since they should apply to this use case.